internal/cloud/gcp/compute.go

```diff
@@ -29,6 +29,18 @@ var GuestOsFeaturesRHEL9 []*computepb.GuestOsFeature = []*computepb.GuestOsFeatu
 	{Type: common.ToPtr(computepb.GuestOsFeature_GVNIC.String())},
 	{Type: common.ToPtr(computepb.GuestOsFeature_SEV_SNP_CAPABLE.String())},
 	{Type: common.ToPtr(computepb.GuestOsFeature_SEV_LIVE_MIGRATABLE_V2.String())},
+	{Type: common.ToPtr(computepb.GuestOsFeature_TDX_CAPABLE.String())},
+}
+
+// Guest OS Features for RHEL9.2 images.
+// The TDX support was added since RHEL-9.6.
+var GuestOsFeaturesRHEL92 []*computepb.GuestOsFeature = []*computepb.GuestOsFeature{
```

Comment:
Nitpick: I think that the comment should state `for RHEL images up to 9.5` and I'd suggest to also name the variable `GuestOsFeaturesRHEL95`

Reply:
Addressed this in the last version.